src/extension.ts

```diff
@@ -42,7 +44,7 @@ let sonarlintOutput: VSCode.OutputChannel;
 let ruleDescriptionPanel: VSCode.WebviewPanel;
 let languageClient: SonarLintExtendedLanguageClient;
 
-function logToSonarLintOutput(message) {
+export function logToSonarLintOutput(message) {
```

Comment:
I wonder whether we should extract logging stuff to a dedicated module also. As it stands now, there is a cyclic dependency between `extension.ts` and `java.ts` - not a big issue, but annoying nonetheless.